pkg/kapis/devops/v1alpha3/handler.go

```diff
@@ -417,14 +417,14 @@ func (h *devopsHandler) DeleteCredential(request *restful.Request, response *res
 
 func (h *devopsHandler) getDevOps(request *restful.Request) (operator devops.DevopsOperator, err error) {
 	ctx := request.Request.Context()
-	token := ctx.Value(constants.K8SToken).(string)
+	token := ctx.Value(constants.K8SToken).(constants.ContextKeyK8SToken)
```

Comment:
The type of `ctx.Value(constants.K8SToken)` should be string instead of `constants.ContextKeyK8SToken`, or it will panic APIServer when we get the DevOps projects.

Reply:
I found it. My bad. I'm going to fix in the later PR.